app/src/main/java/com/omkarmoghe/pokemap/views/MainActivity.java

```diff
@@ -58,20 +55,18 @@ protected void onCreate(Bundle savedInstanceState) {
                 .commit();
     }
 
-    @Override
-    public void onStart(){
-        super.onStart();
-        EventBus.getDefault().register(this);
-    }
 
+    // as discussed with @s7092910, @comann, @christiancoleman: Lets use onPause() / onResume()
     @Override
     public void onResume(){
         super.onResume();
+        EventBus.getDefault().register(this);
     }
 
+    // as discussed with @s7092910, @comann, @christiancoleman: Lets use onPause() / onResume()
```

Comment:
no need for this. 
Just summarize the method, it's history isn't really needed.